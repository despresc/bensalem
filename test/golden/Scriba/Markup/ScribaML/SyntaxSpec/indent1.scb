&p indent
      in a paragraph
  like so
    and more

&p
  indent
    in
   a
     paragraph

&p [ ]   indent
    more

&p [ ]   indent
             more

&p [
  ] indent
       and
      more
