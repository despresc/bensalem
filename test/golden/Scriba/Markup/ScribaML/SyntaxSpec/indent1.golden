[ ElementNode
 ( Element
  { elementTagPos = SrcSpan
   { srcSpanName = ""
   , srcSpanStart = SrcPos
    { srcOffset = 0
    , srcLine = 1
    , srcCol = 1
    }
   , srcSpanEnd = SrcPos
    { srcOffset = 2
    , srcLine = 1
    , srcCol = 3
    }
   }
  , elementName = "p"
  , elementAttrs = Nothing
  , elementArgs =
   [ ElementArg
    { elementArgTy = ElementArgBody
    , elementArgContent =
     [ PlainText "indent"
     , LineEnd
     , LineSpace 4
     , PlainText "in"
     , LineSpace 1
     , PlainText "a"
     , LineSpace 1
     , PlainText "paragraph"
     , LineEnd
     , PlainText "like"
     , LineSpace 1
     , PlainText "so"
     , LineEnd
     , LineSpace 2
     , PlainText "and"
     , LineSpace 1
     , PlainText "more"
     ]
    }
   ]
  }
 )
, LineEnd
, LineEnd
, ElementNode
 ( Element
  { elementTagPos = SrcSpan
   { srcSpanName = ""
   , srcSpanStart = SrcPos
    { srcOffset = 55
    , srcLine = 6
    , srcCol = 1
    }
   , srcSpanEnd = SrcPos
    { srcOffset = 57
    , srcLine = 6
    , srcCol = 3
    }
   }
  , elementName = "p"
  , elementAttrs = Nothing
  , elementArgs =
   [ ElementArg
    { elementArgTy = ElementArgBody
    , elementArgContent =
     [ PlainText "indent"
     , LineEnd
     , LineSpace 2
     , PlainText "in"
     , LineEnd
     , LineSpace 1
     , PlainText "a"
     , LineEnd
     , LineSpace 3
     , PlainText "paragraph"
     ]
    }
   ]
  }
 )
, LineEnd
, LineEnd
, ElementNode
 ( Element
  { elementTagPos = SrcSpan
   { srcSpanName = ""
   , srcSpanStart = SrcPos
    { srcOffset = 95
    , srcLine = 12
    , srcCol = 1
    }
   , srcSpanEnd = SrcPos
    { srcOffset = 97
    , srcLine = 12
    , srcCol = 3
    }
   }
  , elementName = "p"
  , elementAttrs = Just
   ( Located
    { locatedSpan = SrcSpan
     { srcSpanName = ""
     , srcSpanStart = SrcPos
      { srcOffset = 98
      , srcLine = 12
      , srcCol = 4
      }
     , srcSpanEnd = SrcPos
      { srcOffset = 101
      , srcLine = 12
      , srcCol = 7
      }
     }
    , locatedVal = AttrMap
     { unAttrMap = fromList [] }
    }
   )
  , elementArgs =
   [ ElementArg
    { elementArgTy = ElementArgBody
    , elementArgContent =
     [ PlainText "indent"
     , LineEnd
     , LineSpace 1
     , PlainText "more"
     ]
    }
   ]
  }
 )
, LineEnd
, LineEnd
, ElementNode
 ( Element
  { elementTagPos = SrcSpan
   { srcSpanName = ""
   , srcSpanStart = SrcPos
    { srcOffset = 121
    , srcLine = 15
    , srcCol = 1
    }
   , srcSpanEnd = SrcPos
    { srcOffset = 123
    , srcLine = 15
    , srcCol = 3
    }
   }
  , elementName = "p"
  , elementAttrs = Just
   ( Located
    { locatedSpan = SrcSpan
     { srcSpanName = ""
     , srcSpanStart = SrcPos
      { srcOffset = 124
      , srcLine = 15
      , srcCol = 4
      }
     , srcSpanEnd = SrcPos
      { srcOffset = 127
      , srcLine = 15
      , srcCol = 7
      }
     }
    , locatedVal = AttrMap
     { unAttrMap = fromList [] }
    }
   )
  , elementArgs =
   [ ElementArg
    { elementArgTy = ElementArgBody
    , elementArgContent =
     [ PlainText "indent"
     , LineEnd
     , LineSpace 10
     , PlainText "more"
     ]
    }
   ]
  }
 )
, LineEnd
, LineEnd
, ElementNode
 ( Element
  { elementTagPos = SrcSpan
   { srcSpanName = ""
   , srcSpanStart = SrcPos
    { srcOffset = 156
    , srcLine = 18
    , srcCol = 1
    }
   , srcSpanEnd = SrcPos
    { srcOffset = 158
    , srcLine = 18
    , srcCol = 3
    }
   }
  , elementName = "p"
  , elementAttrs = Just
   ( Located
    { locatedSpan = SrcSpan
     { srcSpanName = ""
     , srcSpanStart = SrcPos
      { srcOffset = 159
      , srcLine = 18
      , srcCol = 4
      }
     , srcSpanEnd = SrcPos
      { srcOffset = 164
      , srcLine = 19
      , srcCol = 4
      }
     }
    , locatedVal = AttrMap
     { unAttrMap = fromList [] }
    }
   )
  , elementArgs =
   [ ElementArg
    { elementArgTy = ElementArgBody
    , elementArgContent =
     [ PlainText "indent"
     , LineEnd
     , LineSpace 5
     , PlainText "and"
     , LineEnd
     , LineSpace 4
     , PlainText "more"
     ]
    }
   ]
  }
 )
, LineEnd
]